Marlin/src/gcode/feature/powerloss/M413.cpp

```diff
@@ -48,9 +48,8 @@ void GcodeSuite::M413() {
     if (parser.seen_test('W')) recovery.save(true);
     if (parser.seen_test('P')) recovery.purge();
     if (parser.seen_test('D')) recovery.debug(PSTR("M413"));
-    #if PIN_EXISTS(POWER_LOSS)
-      if (parser.seen_test('O')) recovery._outage();
-    #endif
+    if (parser.seen_test('O')) recovery._outage();
```

Comment:
shouldn't this call like this `recovery._outage(true)` so that the firmware doesn't call `kill()` ?